http-netty/src/main/java/io/micronaut/http/netty/NettyHttpHeaders.java

```diff
@@ -292,8 +295,48 @@ public OptionalLong contentLength() {
         return OptionalLong.empty();
     }
 
+    @Override
+    public List<MediaType> accept() {
+        // use HttpHeaderNames instead of HttpHeaders
+        return MediaType.orderedOf(getAll(HttpHeaderNames.ACCEPT));
+    }
+
+    @Nullable
+    @Override
+    public Charset acceptCharset() {
+        String text = get(HttpHeaderNames.ACCEPT_CHARSET);
```

Comment:
I think this method would be better in `HttpHeaderUtils` with signature 

```java
@Nullable
public static Charset acceptCharset(@Nullable String acceptHttpHeaderValue)
````

That way we could use it in non netty implementations

Reply:
it uses netty-specific apis (HttpHeaderNames) for performance